uni/lib/view/bug_report/bug_report.dart

```diff
@@ -22,4 +22,9 @@ class BugReportPageViewState extends GeneralPageViewState<BugReportPageView> {
 
   @override
   Future<void> onRefresh(BuildContext context) async {}
+
+  @override
+  Container getTopRightButton(BuildContext context) {
+    return Container();
+  }
```

Comment:
Do you need to override `getTopRightButton` while the default implementation already returns a container?

Reply:
Definitely not, I forgot it since I added the base implementation later. Thanks!